### DISCUSSION TITLE
Doesnt reload page if there is not body tag

### DISCUSSION THREAD

**Discussion by amamama45** [BrowserSync/browser-sync#2077] [General]
Hi I spent last 2 days figuring out why it reloads an html pages and why it doesn't express nodejs(without any html). Chatgpt  led me so deep and so wrong, when the problem was so simple. I also spent 1-2 hours reading your docs, and there is not clear description of mechanism how the package injects itself. 
There is many SO threads (i.e https://stackoverflow.com/questions/31163754/browser-sync-does-not-refresh-page-after-changes-with-gulp) with pretty high upvote numbers, comment on YT also saying that many people have some alike problems.
So my suggestion is to upgrade docs with a few articles with explanation of basic mechanism. Or if its possible to check if there is not body tag inject into html

**Comment by shakyShane** [comment]
It's in the 'Requirements' section of the docs page https://browsersync.io/docs.

But yeah, I agree more knowledge of this should be spread. if you wanted to make a start on an 'faq' for the site, feel free and I'll help you get it merged.

**Reply by trymeouteh** [reply]
https://github.com/BrowserSync/browser-sync/discussions/2083
